examples/scripts/counter.js

```diff
@@ -17,7 +17,6 @@
 // * basic properties, actions, events
```

Comment:
I would add a comment like the following:
```js
// This is an example Thing script. 
// It has a count property that can be incremented or decremented via actions  and its changes are reported via events.
// It also has two properties that return an image. The SVG property is also influenced by the increment and decrement actions.
// Features:
// * basic properties, actions, events
// * uriVariables for actions
// * multi-language for descriptions and titles
// * image contentTypes for properties. Note: the contentType applies to all forms of the property
```

Reply:
Makes sense. Done as suggested